cczoo/tensorflow-serving-cluster/tensorflow-serving/docker/secret_prov/sgx_default_qcnl.conf

```diff
@@ -1,5 +1,20 @@
 # PCCS server address
-PCCS_URL=https://pccs.service.com:8081/sgx/certification/v3/
-# # To accept insecure HTTPS cert, set this option to FALSE
-USE_SECURE_CERT=FALSE
+PCCS_URL=https://sgx-dcap-server-tc.bj.tencent.cn/sgx/certification/v3/
```

Comment:
this will change the default pccs url with tc.bj.tencent.cn.
it will not work for other senarios.